change/@fluentui-react-virtualizer-40268eff-578b-4dfe-8ddc-046946884b0e.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "feat: Add automeasuring on dynamic virtualizezr if needed",
```

Comment:
maybe add something about it being the default if no custom measure is provided?